src/app/state/courses/couses.actions.ts

```diff
@@ -0,0 +1,32 @@
+import { createAction, props } from '@ngrx/store';
+import Course from '../../pages/courses-page/components/course/course.types';
+
+export const addCourse = createAction(
```

Comment:
Change actions: 
1. add LoadCoursesList action;
2. add CoursesListLoadedSuccessfully action;
3. add CoursesListLoadedFailed action;
4. remove retrievedCoursesList action;
5. remove retrievedMoreCoursesList action. 
6. remove findCourse;
7. add loadCourseAction, CourseLoadedSuccessfully and CourseLoadedFailed  actions